text/0000-lts-release-strategy.md

```diff
@@ -0,0 +1,90 @@
+
+- Stable and well-tested code
+- API contract stability
+- 3rd digit patch releases for bug and security fixes, delivered for an extended period of time on a regular cadence (or more frequently as needed)
+- Minimal set of feature additions and other changes that can easily be applied, reducing the risk of functional regressions and bugs
+
+Similarly, there are benefits to Hyperledger Fabric maintainers, code contributors, and the wider community:
+
+- New features and other changes can quickly be applied to the master branch, and distributed to the user community for trial, without impacting production deployments.
+- Community feedback on new features can be solicited and acted upon.
+- Bug fixes only need to be backported to a small number of designated LTS releases.
+- Extra tests (e.g. upgrade tests for non-subsequent versions) only need to be executed against a small number of designated LTS releases.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Release cadence
+
+Hyperledger Fabric will continue the pattern of quarterly minor releases, e.g. v2.0, v2.1, v2.2, etc.
+
+## LTS release cadence
+
+Because a new major release typically has large new features that may not yet be tried by the user community, and because SDKs may lag in support of the new release, it is not expected that a new major release will immediately be designated as a LTS release, for example, v2.0 is not a LTS release.
+
+Each major release will eventually have at least one minor release designated by the Fabric maintainers as the LTS release, e.g. v1.4 for v1.x, v2.2 for v2.x (all references to future versions are hypothetical).
+
+After a LTS release is designated, quarterly releases will continue as normal. The Fabric maintainers will decide whether a next release is another minor release, or a new major release. There is no predefined timing for next major version. The decision is typically based on semantic versioning considerations, such as whether API changes are needed or older capabilities need to be removed. Other considerations may also apply, for example significant upgrade steps may motivate a shift to a new major version.
+
+If a major release is not delivered for an extended period of time, the maintainers may designate a subsequent minor release as the next LTS release, for example if v3.1 is a LTS release and there is no need to increment to 4.0 for several quarters, the maintainers may decide to designate v3.4 as the next LTS release.
+
+## LTS 3rd digit patch releases
+
+For LTS releases, 3rd digit patch releases will be provided for bug and security fixes at least every three months, but may be more frequent if important fixes become available sooner. Significant new features and other changes will not be included in 3rd digit patch releases, in order to ensure the stability of the LTS release and reduce the risk of functional regressions and bugs.

```

Comment:
It's hard to imagine that there will be no bug or security fixes within 3 months, but it seems a little odd to commit to putting out a new release, even if there is no content.

Reply:
Changing to:
For LTS releases, 3rd digit patch releases will be provided for bug and security fixes approximately every three months (may be more frequent if important fixes become available sooner, and may be deferred if there are no fixes to publish).